src/main/java/core/basesyntax/controller/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.lib.Injector;
+import core.basesyntax.model.Car;
+import core.basesyntax.service.CarService;
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class GetMyCurrentCarsController extends HttpServlet {
+    private static final String DRIVER_ID = "driverId";
+    private static final Injector INJECTOR = Injector.getInstance("core.basesyntax");
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("core.basesyntax");
```
https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

Reply:
Fixed